charts/prometheus/README.md

```diff
@@ -433,7 +169,7 @@ serverFiles:
 ```
 
 ```console
-helm install stable/prometheus --name my-release -f values.yaml -f service1-alert.yaml -f service2-alert.yaml
+helm install -f values.yaml -f service1-alert.yaml -f service2-alert.yaml ...etc
```

Comment:
Can we include please only working examples - just noticed this one?

Reply:
Fixed in adb4e21